client/Components/EmbedReviewForm/EmbedReviewForm.jsx

```diff
@@ -1,9 +1,90 @@
-import React from " react";
+import React from "react";
+import styles from "./EmbedReviewForm.css";
 
-const EmbedReviewForm = () => {
+const EmbedReviewForm = props => {
+  const rating = props.current[0].ratings;
+  const userStars = props.ratings.count_starRatings;
+  console.log(props.ratings);
   return (
     <div>
-      <h1>Form 1</h1>
+      <h3>Embed This Review</h3>
+      <hr />
+      <div className={styles.formLayout2}>
+        <label htmlFor="embed">
+          Embedding Code
+          <span>
+            Add this review to your website by copying the code below.
+          </span>
+        </label>
+        <input name="embed" type="text" className={styles.inputsForm1} />
+      </div>
+      <button className={styles.smallBtn}>Copy Code</button>
+      <p className={styles.ePreview}>Preview</p>
+      <div className={styles.eReview}>
+        <p className={styles.resName}>{props.current[0].name}</p>
+        <div
+          className={
+            rating === 5
+              ? styles.ratings5
```

Comment:
I think it's not proper to use nested ternary operator, but personally, I prefer it. So no problem here, just commenting.